docs/portal/applications/viewing-application-information.md

```diff
@@ -12,14 +12,14 @@ The following procedure describes how to view information about applications.
 | **Element Number**       | **Description**                               |
 | -------------------------|-----------------------------------------------| 
 | 1                        | Click this button to return to the Applications page.                                                                      |
```

Comment:
In addition to this screen shot, there are 11 other updated ones I captured on 7/22 that have not yet been incorporated. I have refrained from adding snap shots because the ones I capture are of low resolution. Now that I know that  Eleazar is the point of contact for generating high-resolution screen shots (we worked together this week to add refreshed CDN360 screen shots), I will work with him to add high-res screen shots to the ECP help.

Reply:
Please give further comment if any updates. Let's make it complete before approving the PR and merging into master branch.

Reply:
I replaced screen shots with high-res versions.